content/en/th3/500_accessibility/311_screenReaders.md

```diff
@@ -7,28 +7,64 @@ weight: '502'
 
 ## Reading with a screen reader
 
-When launching Thorium, if a screen reader is detected, the relative
-functions are activated.
+### Activating screen reader support 
 
-Note that screen readers use a separate buffer to access the HTML
-content which results in the visual scroll / page offset to be out of
-sync. In other words, Thorium is generally not aware of the user\'s
+Thorium automatically detects assistive technology, but, since 3.3, users must explicitly activate screen reader support by ticking the checkbox in the application settings (this is a global preference for the main library window as well as all reader windows). 
+
+In the previous versions, this support was automatically activated. We had to change because of third party apps creating false positives because they activate assistive technology privileges.
+
+Thorium adapts its behavior when screen reader support is enabled:
+* Text input fields for bookshelf searches require explicit validation (e.g., pressing Enter or clicking the search button) to trigger the search, instead of the default "search as you type" behavior.
+* In the reader window, EPUB HTML documents default to scroll mode instead of paginated presentation to avoid issues with CSS columns.
+* HTML document webviews are fully refreshed on navigation, instead of being reused, to prevent screen readers from losing track of the text.
+* Detection of the user's reading location based on visual scroll offset is disabled to avoid misinterpretations caused by screen reader interaction-induced HTML layout shifts.
+
+### Reading position synchronisation
+
+Since Thorium 3.2, Thorium knows the screen reader reading position and therefore maintains visual scroll synchronisation. In previous versions, because of the screen reader using a separate buffer, 
+Thorium was not aware of the user\'s
 current reading location inside the screen reader buffer, unless the
 screen reader user triggers standard web API events (such as mouse click
 anywhere in the text, or keyboard tab into focusable HTML elements).
 
-Accessibility features addressed to assistive technologies are strongly
-affected by the quality of the book files.
+## Annotating & bookmarking
+
+Based on the feedback we have received so far, it looks like it is very difficult for screen reader users to select an actual range of characters. Such assistive technology allows users to select text inside the screen reader's virtual buffer, so the application isn't aware that the user is reading at a particular location and is selecting text there.
```

Comment:
> to select an actual range of characters

===>

> to select a range of characters inside the actual HTML document (i.e. DOM)

Reply:
Not sure HTML and DOM are understandable information for readers.

Reply:
What about?

> to select a range of characters inside the actual EPUB document (i.e. HTML DOM)